### ISSUE TITLE
SBST.gov 508

### ISSUE THREAD

**Issue by nickbristow** [us-federal-sbst/federalist-modern-team-template#41]
- [x] Image links on home page need content, There is no text or alt tags for these anchors. 
- They should have descriptive text in the `<a> *here </a>` even if its hidden for screen reader users
<img width="709" alt="social_and_behavioral_sciences_team" src="https://cloud.githubusercontent.com/assets/5296248/9886309/9bcba4ae-5bb7-11e5-8619-d026567b8440.png">
- [x] Set the language attribute for the page `<html lang='en'>`
- [x] Remove empty `<h3>` tag in footer, or alternatively, add intended content
- [x] Add Skip Nav link to top of tab order
